ext_emconf.php

```diff
@@ -20,14 +20,11 @@
     'author' => 'Documentation Team',
     'author_email' => 'documentation@typo3.org',
     'state' => 'stable',
-    'uploadfolder' => 0,
-    'createDirs' => '',
-    'clearCacheOnLoad' => 0,
     'author_company' => '',
     'version' => '13.0.0',
     'constraints' => [
         'depends' => [
-            'typo3' => '13.0.0-13.4.99',
+            'typo3' => '13.1.0-13.4.99',
             'impexp' => '13.0.0-13.4.99',
```

Comment:
Please adjust also the other TYPO3 sysexts for consistency.

Reply:
Done